mathml/presentation-markup/operators/mo-form-fallback.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8"/>
+    <title>form fallback</title>
+    <link rel="help" href="https://mathml-refresh.github.io/mathml-core/#tokenmo">
+    <meta name="assert" content="Verify fallback to postfix/prefix forms.">
+    <link rel="match" href="mo-form-fallback-ref.html">
+  </head>
+  <body>
+
+    <!-- The left (respectively right) parenthesis only has a prefix
+         (respectively postfix) form which gives its stretchiness.
+         In theory, the parenthesis in the middle of the <math> element are
+         considered infix operators and should not stretch. Let's check that we
+         fallback to the postfix/prefix forms and actually make them stretchy.
+         This is to handle bad MathML markup that misses explicit <mrow> tags.
+      -->
```

Comment:
I think we need to open an issue in the CG tracker as I don't think that heuristic is standard. Probably the form stuff is only in mathml full not mathml core yet.